menu.css

```diff
@@ -178,19 +180,41 @@ ul a:active{color: black;} /* Selected link */
 .nav li a{
 	float: right;
 	font-size: 16px;
-	color: black;
+	color: white;
 	text-align: left;
 	padding: 14px;
 	text-decoration: none;
 	overflow: none;
 	z-index: 4;
-	font-family: 'Poppins', sans-serif;
-	background-color: red;
+	font-family: 'Lato', sans-serif;

```

Comment:
font and background colour changed for nav button for the first colours and fonts design